tile/tile.yml

```diff
@@ -1,13 +1,18 @@
 ---
-name: datadog-application-monitoring # By convention lowercase with dashes
+name: datadog_application_monitoring # By convention lowercase with underscores
```

Comment:
to remove the deprecation warning `DeprecationWarning: datadog_application_monitoring expected, got datadog-application-monitoring !!!`

Reply:
Yeah sgtm

Reply:
Yeah sgtm